src/tooltip/tooltip.js

```diff
@@ -276,6 +284,14 @@ angular.module( 'ui.bootstrap.tooltip', [ 'ui.bootstrap.position', 'ui.bootstrap
               ttScope.title = val;
             });
 
+            function watchNgDisabled() {
+              unbindWatcher = scope.$watch(attrs.ngDisabled, function (newValue, oldValue) {
```

Comment:
Is there any reason why you cannot do `attrs.$observe('disabled', ...)` here or something similar?